packages/ampd-sdk/src/grpc/client.rs

```diff
@@ -24,11 +30,13 @@ pub trait Client {
     // TODO: This trait's methods should return our own types rather than the generated protobuf ones
     async fn subscribe(
         &self,
-        request: SubscribeRequest,
-    ) -> Result<Streaming<SubscribeResponse>, Error>;
+        filters: Vec<Event>,
+        include_block_begin_end: bool,
+    ) -> Result<Streaming<Event>, Error>;
```

Comment:
I don't want it to return `Streaming` but an associated generic type bounded by `Stream`. It makes the mock much easier to work with.